pennylane/tape/tapes/qubit_param_shift.py

```diff
@@ -169,7 +169,13 @@ def processing_fn(results):
                 array[float]: 1-dimensional array of length determined by the tape output
                 measurement statistics
             """
-            return np.dot(coeffs, np.squeeze(results))
+            results = np.squeeze(results)
+
+            if results.dtype is not np.dtype("O"):

```

Comment:
For institutional memory, would you mind dropping a comment on what case is caught here?

Reply:
Sure!